unittest/formats/test_atom_styles.cpp

```diff
@@ -3013,437 +3014,9 @@ TEST_F(AtomStyleTest, template)
     ASSERT_EQ(lmp->atom->nghost, 0);
     ASSERT_NE(lmp->atom->nmax, -1);
     ASSERT_EQ(lmp->atom->tag_enable, 1);
-    ASSERT_EQ(lmp->atom->q_flag, 0);
```

Comment:
These tests must be re-instated. see my previous comments.

Reply:
done